src/app.js

```diff
@@ -25,11 +36,102 @@ const quoteData = [
   },
 ];
 
+// const renderList = (quoteList) => {
+//   const $quoteList = $();
+//   $quoteList.empty();
+//
+//   quoteList.forEach((quote) => {
+//     const quoteView = new QuoteView({
+//       model: quote,
+//       template: _.template($('#quote-template').html()),
+//       tagName: 'li',
+//       className: 'quote',
+//     });
+//     $quoteList.append(quoteView.render().$el);
+//   });
+// };
+
+let quoteTemplate;
+let tradeTemplate;
+let orderTemplate;
+const bus = _.extend({}, Backbone.Events);
+
 $(document).ready(function() {
+  // Quotes
   const quotes = new QuoteList(quoteData);
   const simulator = new Simulator({
     quotes: quotes,
   });
 
   simulator.start();
+
+  quoteTemplate = _.template($('#quote-template').html());
+
+  const quoteListView = new QuoteListView ({
+    model: quotes,
+    template: quoteTemplate,
+    el: 'main',
+  });
+
+  quoteListView.render();
+
+  // Trade History
+  tradeTemplate = _.template($('#trade-template').html());
+
+  const tradeHistoryView = new TradeHistoryView({
+    model: quotes,
+    template: tradeTemplate,
+    el: 'main',
+
+  });
+
+  tradeHistoryView.bind();
+
+  // Add orders
+  const orders = new OrderList();
+
+  $('#orderForm').on('submit', function(event) {
+    event.preventDefault();
+  });
+
+  // quotes.each((quote) => {
+  //   $('#orderForm select').append($('<option>', {
+  //     value: quote.get('symbol'),
+  //     text: quote.get('symbol')
+  //   }));
+  // });
+
+  const orderEntryView = new OrderEntryView({
+    el: '.order-entry-form',
+    bus: bus,
+    quotesList: quotes
```

Comment:
I very much like that you pass the quote list directly to the `OrderEntryView`. Building this connection (as opposed to using the event bus or something else) will make your program's design much simpler.